mmf/models/mmbt.py

```diff
@@ -482,27 +473,42 @@ def __init__(self, config, *args, **kwargs):
         self.config = config
         self.bert = MMBTBase(config, *args, **kwargs)
         self.encoder_config = self.bert.encoder_config
+        self.num_labels = self.config.num_labels
 
         self.dropout = nn.Dropout(self.encoder_config.hidden_dropout_prob)
         self.classifier = nn.Sequential(
             BertPredictionHeadTransform(self.encoder_config),
             nn.Linear(self.encoder_config.hidden_size, self.config.num_labels),
         )
 
-    def forward(self, sample_list):
-        module_output = self.bert(sample_list)
+    def forward(
+        self,
+        input_modal: Tensor,
+        input_ids: Tensor,
+        segment_ids: Tensor,
+        input_mask: Tensor,
+        modal_token_type_ids: Optional[Tensor] = None,
+    ):
+        module_output = self.bert(
+            input_modal=input_modal,
+            input_ids=input_ids,
+            segment_ids=segment_ids,
+            input_mask=input_mask,
+            modal_token_type_ids=modal_token_type_ids,
+        )
         pooled_output = module_output[1]
         output = {}
 
-        if (
-            self.encoder_config.output_hidden_states
-            or self.encoder_config.output_attentions
-        ):
-            output["extras"] = module_output[2:]
+        if not torch.jit.is_scripting():
+            if (
```

Comment:
Some comments from other PR are also applicable here, please take care of those.